frontend/src/components/tickets-page/review/create-order-buttons.tsx

```diff
@@ -40,7 +40,7 @@ export const CreateOrderButtons: React.SFC<Props> = ({ state, email }) => {
       )
         .flat()
         .map((product) => ({
-          ticketId: product.id.replace("-en", "").replace("-it", ""),
+          ticketId: product.id,
```

Comment:
shouldn't add the language here?

Reply:
this is the mapping to send the ticket id to createOrder, we only care about the ID you are purchasing, nothing else